doc/FEATURES_RELEASE_2.md

```diff
@@ -37,10 +38,14 @@ interaction windows.
 ### Testing
 
 Visit the [homepage]. Select `panacea/test/fixtures/ddis.pml` which contains a
-DDI. Press `Submit`. The system should display the DDI information
-for the DDI found, including whether its disposition is `Good` or `Bad`,
-the timing window and units of this timing window, as well as the drugs involved.
+DDI. Press `Submit`. The system should display the DDI information for the DDI
```

Comment:
In the product backlog this feature refers only to CSV generation. We should probably link to the CSV export section and then outline talk about how that data is displayed to the user.

To explain `Good/Bad` vs `Harmful/Not Harmful` -  perhaps we should include something mentioning:

1) The terminology he asked for (`Good`/`Bad`) is used in the CSV
2) The terminology we display to a clinician (`Harmful`/`Not harmful`) was chosen because it is more suitable for the use case. `harmful` for the patient may not necessarily be `bad`. Some treatments are `harmful` but overall have a `good` effect.